Jesus/binaries

TheYisusGithub opened the pull request:
Hi team, this is my code and it is ready for review.

TheYisusGithub committed:
I write the code for ConvertToDecimalInt

TheYisusGithub committed:
I get out funtions that I don't use

J0c4 commented on src/test/java/binaries/jesus/BinaryArrayTest.java:
```diff
@@ -0,0 +1,7 @@
+package binaries.jesus;
```
No need to add test class yet, please remove it for now.

J0c4 commented on src/main/java/binaries/jesus/BinaryArray.java:
```diff
@@ -18,6 +18,37 @@ public BinaryArray(int[] binaries) {
+
+    public  double [] pow(int arrayBinary[]){
+        double [] arrPow= new double[arrayBinary.length];
+        int suma=0;
```
spaces missing before and after `=`. Please also check this in other parts of your code and fix.

J0c4 commented on src/main/java/binaries/jesus/BinaryArray.java:
```diff
@@ -18,6 +18,37 @@ public BinaryArray(int[] binaries) {
+
+    public int sumPow(int [] arrPow,int arrBinary[]){
+        int sum=0;
+        for (int i = 0; i <arrBinary.length ; i++) {
```
spaces

J0c4 commented on src/main/java/binaries/jesus/BinaryArray.java:
```diff
@@ -18,6 +18,37 @@ public BinaryArray(int[] binaries) {
      */
     public int ConvertToDecimalInt() {
-        throw new UnsupportedOperationException("Not supported yet.");
+        //throw new UnsupportedOperationException("Not supported yet.");
```
please remove commented line

J0c4 commented on .idea/workspace.xml:
```diff
@@ -0,0 +1,262 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
please remove IDE's generated files by running `git rm -r .idea`

J0c4 commented:
Hi @TheYisusGithub , please don't forget to address my comments and remove the unneeded files that still present in this PR, so you don't get delayed in the next assignment. If you need help just let me know.

TheYisusGithub committed:
implement reverse
Revert "I get out funtions that I don't use"

This reverts commit d3bc0d1bf1417491a9151371fd6f96132a303de9.

TheYisusGithub committed:
Revert "implement reverse again"

This reverts commit 610144641d401f2f1426a7cec6e0faa11968cd82.

TheYisusGithub committed:
Erase folder idea

TheYisusGithub committed:
Correct some fix and implement test cases.

TheYisusGithub committed:
Implement try and cach and using some exceptions to my test

TheYisusGithub committed:
Correct some thing about throw and i comment some test.
I erase some  system.out.println and replace with throw in the sentences try and catch and comment 2 test cases that dont run.
